user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_TakeConfigurationBackup.md

```diff
@@ -0,0 +1,200 @@
+
+## Prerequisites
+
+- The DataMiner System used for this tutorial has to meet the following requirements:
+
+  - The DataMiner System is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+  - The DataMiner version is 10.3.0 [CU0] or higher.
+
+  - The latest version of IDP is installed.
+
+    > [!TIP]
+    > See [Installing DataMiner IDP](xref:Installing_DataMiner_IDP)
+
+- A shared folder must be accessible from the system to store the backups.
+
+> [!IMPORTANT]
+> Do not use a DaaS system to follow this tutorial. On DaaS systems, it is not possible to configure a shared folder or access a shared folder outside the DaaS system.
+
+## Overview
+
+- [Step 1: Create the CI Type](#step-1-create-the-ci-type)
+- [Step 2: Create an element and manage it](#step-2-create-an-element-and-manage-it)
+- [Step 3: Set up the configuration archive](#step-3-set-up-the-configuration-archive)
+- [Step 4: Create the script to retrieve the configuration backup](#step-4-create-the-script-to-retrieve-the-configuration-backup)
+- [Step 5: Retrieve the device configuration](#step-5-retrieve-the-device-configuration)
+
+## Step 1: Create the CI Type
+
+First, a [CI Type](xref:CI_Types) needs to be created for IDP to know how to manage the element or device.
+
+Instead of creating the CI Type from scratch, you will generate it based on an existing connector. In this example, the [Generic Switch](https://catalog.dataminer.services/details/connector/931) connector is used, but you can use any other connector.
+
+1. Deploy the connector to start from:
+
+   1. Deploy the latest version of the [Generic Switch](https://catalog.dataminer.services/details/connector/931) connector from the DataMiner Catalog.
+
+      > [!TIP]
+      > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
+
+   1. Make sure the latest version is [set as production](xref:Promoting_a_protocol_version_to_production_version).
+
+      ![Generic Switch connector](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_ProtocolTemplates.png)
+
+1. Generate the CI Type:
+
+   1. Open the IDP app.
+
+   1. Go to *Admin* > *CI Types*.
+
+   1. Above the table on the right, click *Generate* to start the wizard.
+
+      ![Start to generate the CI Type script](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_0.png)
+
+   1. In the wizard, select the connector and click *Generate*.
+
+      ![Select the connector](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_1.png)
+
+   1. Click *Finish* to close the wizard.
+
+   1. Switch off the *Show All* button in the top-right corner to locate the generated CI Type more easily.
+
+      ![Generate the CI Type](~/user-guide/images/IDP_Tutorial_TakeConfigurationBackup_GenerateCiType_2.png)
+
+## Step 2: Create an element and manage it
+
+1. [Create an element](xref:Adding_elements) that uses the connector you deployed in the previous step.
```

Comment:
Should we add more information about how to create this element? Specifically, users might not know which IP address to use for this.